services/QuillLMS/spec/requests/secure_headers_spec.rb

```diff
@@ -27,7 +27,7 @@ def index
 
   it 'should set cache control headers' do
     get '/dummy'
-    expect(response.header['Cache-Control']).to match('no-cache, no-store, max-age=0, must-revalidate')
+    expect(response.header['Cache-Control']).to match('no-cache, no-store')
```

Comment:
Note, we are setting this [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/controllers/application_controller.rb#L124). So Rails is stripping it out, but we also not set it to avoid confusion in the future. This might be worth pinging @happythenewsad to give a once over since he worked on this.